src/modules/Packages.rb

```diff
@@ -17,11 +17,15 @@ class PackagesClass < Module
     # All known types of resolvables
     RESOLVABLE_TYPES = [:product, :patch, :package, :pattern, :language]
     # Minimum set of packages required to enable VNC server
-    VNC_BASE_PACKAGES = ["xorg-x11", "xorg-x11-Xvnc", "xorg-x11-fonts", "xinetd"]
+    VNC_BASE_TAGS = ["xorg-x11", "xorg-x11-Xvnc", "xorg-x11-fonts", "xinetd"]
+    # Additional packages needed to run second stage in graphical mode
+    # Use exact package name (i.e. libyui-qt6 instead of libyui-qt) to prevent
+    # the UI from asking for confirmation
+    AUTOYAST_X11_TAGS = ["libyui-qt", "yast2-x11"]
     # Default window manager for VNC if none is installed
     DEFAULT_WM = "icewm"
     # Minimum set of packages required for installation with remote X11 server
-    REMOTE_X11_BASE_PACKAGES = [ "xorg-x11-server", "xorg-x11-fonts", "icewm" ]
+    REMOTE_X11_BASE_TAGS = [ "xorg-x11-server", "xorg-x11-fonts", "icewm" ]
```

Comment:
@ancorgs Yes, that's what I meant (sorry if that was not clear...). I'm not sure if changing `REMOTE_X11_BASE_PACKAGES` to `REMOTE_X11_BASE_TAGS` is safe...

I mean the package list has never caused any issue (at least I'm not aware of any), changing the behavior might break something. Although it's very unlikely there is a problem with multiple providers which is not handled nicely currently.

Selecting the first found in the list is not reliable (the order is not guaranteed) and this can result in "random" package selection.

I know it's not a problem now, but I know that some AutoYaST users are really creative...

In general I'm not against using the provides, but we have to better handle the corner cases, the current implementation is little bit fragile and non-deterministic ....